File: ios/ReactAirplay/Views/RAAirplayButtonViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE (RAAirplayButtonViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor *)
RCT_EXPORT_VIEW_PROPERTY(activeTintColor, UIColor *)
RCT_EXPORT_VIEW_PROPERTY(prioritizesVideoDevices, BOOL)

@end

File: ios/ReactAirplay/Views/RAAirplayButtonViewManager.swift
import AVKit
import React

@available(iOS 11.0, *)
@objc(RAAirplayButtonViewManager)
class RAAirplayButtonViewManager: RCTViewManager {
  override func view() -> (RAAirplayButtonView) {
    return RAAirplayButtonView()
  }

  @objc override class func requiresMainQueueSetup() -> Bool {
    return true
  }
}
